bin/bundle

```diff
@@ -11,7 +11,7 @@ m = Module.new do
   end
 
   def cli_arg_version
-    return unless invoked_as_script? # don't want to hijack other binstubs
+    return unless invoked_as_script? # do not want to hijack other binstubs
```

Comment:
Are we sure we want to edit this file ?

Reply:
Why not?

Reply:
I was thinking that is something we do not really own ...